### ISSUE TITLE
Hard to read language selector

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla/addons-frontend#3205]
Steps to reproduce:
1. Load AMO-dev homepage https://addons-dev.allizom.org/en-US/firefox/
2. Go to page footer and observe language selector

Expected results:
The language selector is clear and readable.

Actual results:
The language is written using black on gray background with icon and drop-down arrow in white.

Notes/Issues:
Maybe we can use white background? Or white color font for language as in production.
Verified on AMO-dev FF55(Win 7). 
Screenshot for this issue:
![2017-09-22_0939](https://user-images.githubusercontent.com/15685960/30732196-087039d2-9f7a-11e7-9f42-fdac31330403.png)

**Comment by tofumatt** [comment]
Yeah, this is hard to read. I say we change our text to white and file an issue upstream as well.

Matthew Riley MacPherson (sent from mobile)

**Comment by muffinresearch** [comment]
I changed the text from white, since that was even worse contrast - thinking the black should be darker. We could also drop back the grey. 

Failing that we should just drop all the styling and go with a regular select which is what mozilla.org does.

**Comment by tofumatt** [comment]
Oh, they use a regular select? Yes, let’s please do that. 😊

The custom one is a bit of a pain to implement and probably less obvious than a regular select against such a dark background.

Matthew Riley MacPherson (sent from mobile)

**Comment by muffinresearch** [comment]
I'm on it.

**Comment by ValentinaPC** [comment]
Verified as fixed on AMO-dev FF55(Win 7, Android 7.2.1).
Postfix screenshot:
![2017-09-22_1432](https://user-images.githubusercontent.com/15685960/30743268-939bf1c4-9fa5-11e7-90c4-2e1bc62355b9.png)
